docs/design/storage-node-logger.md

```diff
@@ -0,0 +1,30 @@
+# Rotating Logger for Storage Node
+
+## Abstract
+
+Storage Node needs a logger that can log to disk, rotate files, compress old files, delete really old files, and ensure a limit on the size of logs.
+
+## Background
+
+Currently log files are agglomerating and are completely unmanaged. Storage nodes need to be able to keep the log files under control to prevent logs from using an exorbitant amount of the storage node's hard drive space.
+
+## Design
+
+Logger Service that runs at storage node startup
+* Write files to disk when log function is called.
+* Rotate the current file being used for logging by current date and log file size.
+* Compress log files older than 3(?) days.
+* Delete compressed files older than 30(?) days.
+* Limit Log files to 10Kb(?).
+
+We can potentially use [natefinch/lumberjack](https://github.com/natefinch/lumberjack) to maintain/rotate/compress/delete logs
```

Comment:
Add link to https://github.com/uber-go/zap/blob/master/FAQ.md#does-zap-support-log-rotation to show how to integrate.